src/write.c

```diff
@@ -660,10 +664,12 @@ static avifResult avifEncoderWriteColorProperties(avifRWStream * outputStream,
     }
 
     // HEIF 6.5.5.1, from Amendment 3 allows multiple colr boxes: "at most one for a given value of colour type"
-    // Therefore, *always* writing an nclx box, even if an a prof box was already written above.
+    // Therefore, *always* writing an nclx box, even if a prof box was already written above.
     AVIF_CHECKRES(avifEncoderWriteNclxProperty(dedupStream, outputStream, imageMetadata, ipma, dedup));
 
-    return avifEncoderWriteExtendedColorProperties(dedupStream, outputStream, imageMetadata, ipma, dedup);
+    AVIF_CHECKRES(avifEncoderWritePasp(dedupStream, outputStream, imageMetadata, ipma, dedup));
```

Comment:
Nit/optional: Name this funciton `avifEncoderWritePaspProperty` to be more similar to `avifEncoderWriteNclxProperty` in naming convention.